web/src/beta/features/PluginPlayground/Plugins/presets/custom/myPlugin.ts

```diff
@@ -33,8 +39,30 @@ const widgetFile: FileType = {
 \`); `
 };
 
+const demoFirstInfoboxFile: FileType = {
+  id: "custom-my-plugin-demo-first-infobox",
+  title: "demo-first-infobox.js",
+  sourceCode: `reearth.ui.show(\`
+  ${PRESET_PLUGIN_COMMON_STYLE}
+  <div id="wrapper">
+    <h2 style="text-align: center;">First Infobox</h2>
+  </div>  
+\`); `
+};
+
+const demoSecondInfoboxFile: FileType = {
+  id: "custom-my-plugin-demo-second-infobox",
+  title: "demo-second-infobox.js",
+  sourceCode: `reearth.ui.show(\`
+  ${PRESET_PLUGIN_COMMON_STYLE}
+  <div id="wrapper">
+    <h2 style="text-align: center;">Second Infobox</h2>
+  </div>  
+\`); `
+};
+
 export const myPlugin: PluginType = {
   id: "my-plugin",
   title: "My Plugin",
-  files: [widgetFile, yamlFile]
+  files: [widgetFile, yamlFile, demoFirstInfoboxFile, demoSecondInfoboxFile]
```

Comment:
```suggestion
  files: [widgetFile,  demoFirstInfoboxFile, demoSecondInfoboxFile, yamlFile]
```
Let's reorder them making scripts together. BTW please update the names as well.

Reply:
done